frontend/vue/components/CodeExercise/CodeOutput.vue

```diff
@@ -78,6 +78,7 @@ export default defineComponent({
       return code.includes('IBMQ.')
     },
     requestExecute (code: string) {
+
```

Comment:
if we remove the extra blank lines in this file then we would not need to commit any changes to the file.